app/DoctrineMigrations/Version20241031130556.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241031130556 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE task ADD co2_emissions INT NOT NULL DEFAULT 0');
```

Comment:
Sorry to ask this question because the whole thing is built around this concept but... do we really need to store this in database, can't we calculate this on the fly?

Reply:
I d rather save it in the DB so we have the data right there to export with the SQL query + be able to check things if we spot mistakes in the calculus

do you have an idea on how it will look if we calculate things on the fly?